spatial/transform/umeyama_test.go

```diff
@@ -0,0 +1,160 @@
+// Copyright ©2025 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package transform
+
+import (
+	"math"
+	"testing"
+
+	"gonum.org/v1/gonum/mat"
+)
+
+func TestUmeyama(t *testing.T) {
+	tol := 1e-10
+
+	testCases := []struct {
+		name      string
+		src       *mat.Dense
+		dst       *mat.Dense
```

Comment:
```suggestion
		from      *mat.Dense
		to        *mat.Dense
```
`src` and `dst` have Go semantic loading that should be avoided.